app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}}}")
+
+
+def create_animal(alive_list: list) -> list:
+    Animal.alive = []
+    for beast in alive_list:
+        animal = Animal(beast["name"], beast["health"], beast["hidden"])
+        Animal.alive.append(animal)
```

Comment:
The line `Animal.alive.append(animal)` is redundant because the `Animal` constructor already appends the instance to `Animal.alive`. You can remove this line to avoid duplication.